share/spack/setup-env.sh

```diff
@@ -255,8 +260,15 @@ fi;
 #
 # set module system roots
 #
-_spack_pathadd DK_NODE    "${_sp_dotkit_root%/}/$_sp_sys_type"
-_spack_pathadd MODULEPATH "${_sp_tcl_root%/}/$_sp_sys_type"
+IFS=':' read -ra tcl_roots <<< "$_sp_tcl_roots"
+for tcl_root in "${tcl_roots[@]}"; do
+    _spack_pathadd MODULE_PATH "${tcl_root}/$_sp_sys_type"
```

Comment:
I' ve tested this PR with boostrap for environment modules and found that MODULEPATH env var is missing.
I see that you have substituted with MODULE_PATH.
Is this intended or just a typo?
In case of typo, do you mind to fix so to ease testing?